### PR TITLE
Cleanup tracing spans

### PR DESCRIPTION
**Pull request by aledbf**
Changes:
- remove unnecessaries changes of types
- remove binary content fields
- send error messages instead of error

### CODE REVIEW

File: components/ws-daemon/pkg/content/initializer.go [hunk 1]
```diff
@@ -98,12 +98,10 @@ func collectRemoteContent(ctx context.Context, rs storage.DirectAccess, ps stora
 
 // RunInitializer runs a content initializer in a user, PID and mount namespace to isolate it from ws-daemon
 func RunInitializer(ctx context.Context, destination string, initializer *csapi.WorkspaceInitializer, remoteContent map[string]storage.DownloadInfo, opts RunInitializerOpts) (err error) {
-	//nolint:ineffassign
+	//nolint:ineffassign,staticcheck
 	span, ctx := opentracing.StartSpanFromContext(ctx, "RunInitializer")
 	defer tracing.FinishSpan(span, &err)
 
-	span.LogKV("remoteContent", remoteContent)
```
Comment (csweichel) [inline components/ws-daemon/pkg/content/initializer.go:105]: Why remove this?

File: components/registry-facade/pkg/registry/manifest.go [hunk 2]
```diff
@@ -219,7 +219,6 @@ func (mh *manifestHandler) getManifest(w http.ResponseWriter, r *http.Request) {
 		}
 
 		dgst := digest.FromBytes(p).String()
-		span.LogKV("manifest", string(p))
```
Comment (csweichel) [inline components/registry-facade/pkg/registry/manifest.go:222]: Why remove this? 
it's been rather handy at times

Comment (aledbf) [reply components/registry-facade/pkg/registry/manifest.go:222]: > it's been rather handy at times

Do we need that in the tracing? Can we replace that with a debug log?

**Review by csweichel — APPROVED** [review]
As discussed offline, the span log removal is an attempt to reign in the tracing memory leak

**Pull request merged**